recipe-app-code/css/main.css

```diff
@@ -43,6 +43,7 @@ body {
   flex-direction: column;
   justify-content: center;
   align-items: center;
+  overflow-x: hidden;
```

Comment:
Good job, @Beg92!  But this needs to be moved into the body section in _general-layout.scss